rfc002-procope-blog.md

```diff
@@ -0,0 +1,208 @@
+
+This proposal also doesn't cover the potential for other media like **podcasts**, but that could be a future expansion.
+
+This proposal also doesn't cover social media, but is certainly tethered to that element of community engagement.
+
+#### Aside: defining white papers vs blog posts
+
+In this model, **blog posts** are community contributions to general knowledge about technical communication, an informal stance of Good Docs Project template support, or other points of interest for those interested in GDP's mission and broader content.
+
+They're periodical content, to keep engagement up. They're individually credited, and live on a GDP-branded Medium.com publication.
+
+To contrast, **white papers** would cover topics core to Good Docs Project templates, in philosophy and execution. They would be created by community consensus and represent the community as a unified element. They would live on the core GDP website, rather than in the publication models outlined below.
+
+White papers are created as needs arise, rather than an ongoing periodical model to keep external engagement up.
+
+The Procope team will be very happy to be involved with white papers, including co-running it to lend editorial expertise, but doesn't want to muddy this proposal by including a different sort of publication.
+
+#### Out of scope for now: templates for this content
+
+Templates are great! That's why GDP exists. But rather than starting with blog post templates, we want to give freedom for various forms of conversational writing.
+
+That said, some months in we do want to examine our content for templating, as that empowers onboarding contributing.
+
+### Content types
+
+All media is community-contributed pieces on documentation philosophy, best practices, and related opinion or research pieces. They are individually created and driven, representing a viewpoint within the community that the community respects enough to be associated with it, but with bylines that attach the viewpoint to the original author or authors.
+
+(This means the authors and the community co-own any potential fallout from poorly received posts or videos.)
+
+Blog posts are 500- to 1000-word pieces about a subject, whether that's a single topic, a top N post, a collection of interesting links with commentary, and so on. Some posts will be solidary, and others will be part of a series.
+
+Videos are 3- to 10-minute pieces about a subject, illustrating a narrative that helps those more engaged in visual narratives buy into our concepts and templates.
+
+#### Where content would live
+
+Blog posts would live on a Medium.com magazine, branded as part of the Good Docs Project. Medium.com is a known place where people can subscribe to stories, comment, and otherwise have a sense of expectation regarding a look and feel. I've taken the liberty of (creating a blank publication)[https://medium.com/the-good-docs-project], to make sure we have the name.
```

Comment:
Around here is where I should add the required features for a blog platform (and what is a nice-to-have or not needed at all, as relevant), based on the outcomes of comments.